pkg/controller/tuned/tuned_controller.go

```diff
@@ -69,7 +69,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 	createCustomResource(mgr)
 	if err != nil {
-		log.Printf("createCustomResource(): %v\n", err)
+		log.Printf("createCustomResource(): %v", err)
```

Comment:
Were there changes to the `log.Printf` to add a new line?

Reply:
Changes?  Not sure it changed, but the documentation clearly states the '\n' is added if it is not present.  And it definitely makes sense for logging.